content/300-guides/050-database/100-developing-with-prisma-migrate/150-team-development.mdx

```diff
@@ -23,6 +16,13 @@ To incorporate changes from collaborators:
 
 Migrations are **applied in the same order as they were created**. The creation date is part of the migration subfolder name - for example, `20210316081837-updated-fields` was created on `2021-03-16-08:08:37`.
 
+<Admonition type="info">
+
+This guide **does not apply for MongoDB**.<br />
+Instead of `migrate dev`, [`db push`](/concepts/components/prisma-migrate/db-push) is used for [MongoDB](/concepts/database-connectors/mongodb).
+
+</Admonition>
```

Comment:
Point out that Migrate for MongoDB is `db push` instead of "not supported" warning.